apps/tasks/lib/mix/tasks/app/version.ex

```diff
@@ -0,0 +1,8 @@
+defmodule Mix.Tasks.App.Version do
+  @moduledoc false
+  def run(_) do
+    Mix.Project.config()
+    |> Keyword.get(:version)
+    |> IO.puts()
```

Comment:
Is this worth making a Logger call?

Reply:
No, the point of the task is to write the app version to your console. If it put the log level in there too, we'd have to parse that piece.